src/main/java/com/contentful/java/cma/model/CMAWebhookTransformation.java

```diff
@@ -25,7 +25,7 @@
   private String method;
   private String contentType;
   private Boolean includeContentLength;
-  private String body;
+  private Object body;
```

Comment:
Contentful creates an object body, not a string
This is a hotfix which needs to be reworked later